app/services/container_dashboard_service.rb

```diff
@@ -160,4 +161,44 @@ def heatmaps
       }
     }
   end
+
+  def node_utilization
+    resource_ids = @ems.present? ? [@ems.id] : ManageIQ::Providers::ContainerManager.all.pluck(:id)
+    metrics = VimPerformanceDaily.find_entries({:tz => @controller.current_user.get_timezone})
+    metrics = metrics.where(:resource_type => 'ExtManagementSystem', :resource_id => resource_ids)
+    metrics = metrics.where("timestamp > ?", 30.days.ago)
```

Comment:
> @zeari @alongoldboim any added value in keeping all the where clause in the same call? E.g.:

When I spilt the line I assumed that since this query executes only at line 177 and that I can form it anyway I like before that